### PR TITLE
Remove submit handler for removed function.

### LINKED ISSUE #6518: Fatal error when trying to save theme settings
#### Current Behavior

Trying to save theme settings results in a fatal error:

```
Fatal error: Call to undefined function paraneue_dosomething_theme_settings_handle_files() in /var/www/dev.dosomething.org/html/includes/form.inc on line 1519
```
#### Desired Behavior

It should not result in a fatal error.
#### Steps to Replicate

Go to the theme settings and try to save them.
#### Why This Matters

Fatal errors are bad.

### PR DESCRIPTION
**Pull request by DFurnes**
#### What's this PR do?

Fixes #6518 by removing the form submit handler for a function removed in #6494.
#### How should this be reviewed?

Try to save the theme settings, you should not get a fatal error.
#### Any background context you want to provide?

💣 🙅 
#### Relevant tickets

Fixes #6517.
#### Checklist
- [ ] Documentation added for new features/changed endpoints.

### CODE REVIEW

File: lib/themes/dosomething/paraneue_dosomething/theme-settings.php [hunk 1]
```diff
@@ -42,11 +42,6 @@ function paraneue_dosomething_form_system_theme_settings_alter(&$form, &$form_st
   _paraneue_dosomething_theme_settings_footer($form, $form_state);
   _paraneue_dosomething_theme_settings_user($form, $form_state);
 
-  if (!isset($form['#submit'])) {
-    $form['#submit'] = [];
```
Comment (angaither) [inline lib/themes/dosomething/paraneue_dosomething/theme-settings.php:46]: does this part also wanna be gone?

Comment (DFurnes) [inline lib/themes/dosomething/paraneue_dosomething/theme-settings.php:46]: We don't use that array anywhere else that I can tell (although who knows 😬).

Comment (angaither) [inline lib/themes/dosomething/paraneue_dosomething/theme-settings.php:46]: I mean as long as it submits and saves values, I'm 🆒

Comment (DFurnes) [inline lib/themes/dosomething/paraneue_dosomething/theme-settings.php:46]: Livin' dangerously! 💣

**Pull request merged**